extensions/podman/packages/extension/src/extension.spec.ts

```diff
@@ -2203,10 +2205,6 @@ describe('registerOnboardingRemoveUnsupportedMachinesCommand', () => {
       command: 'command',
     });
 
-    // two times false (no qemu folders)
-    vi.mocked(fs.existsSync).mockReturnValueOnce(false);
-    vi.mocked(fs.existsSync).mockReturnValueOnce(false);
```

Comment:
The test is design for window, but was mocking mac condition call because the isMac was default to true